inc/customizer/site-identity.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Additional site identity customizer options.
+ *
+ * @package Primer
+ * @since   1.4.2
+ */
+
+class Primer_Site_Identity_Options {
+
+	/**
+	 * Class constructor.
+	 */
+	public function __construct() {
+
+		/**
+		 * Disable additional site identity options
```

Comment:
@EvanHerman Let's tweak this description a little to match how we describe other filters, like [this one](https://github.com/godaddy/wp-primer-theme/blob/f32e224ec0b67371be20db9a6ea94f1c99d92d9c/templates/parts/credit.php#L20-L27).

```
/**
 * Filter the site identity settings display.
 *
 * @since 1.4.2
 *
 * @var bool
 */
```